src/api/structure/interfaces.ts

```diff
@@ -0,0 +1,30 @@
+export interface Metadata {
```

Comment:
I suppose it will be more preferable
``` suggestion
import {Dictionary} from '../../configuration/interfaces';
export type Metadata = Dictionary<string>;
```